### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abjorck**
GoogleAuth usage example in readme outdated.

### CODE REVIEW

**Comment by abjorck** [comment]
I signed it!

File: packages/firebase_auth/README.md [hunk 1]
```diff
@@ -63,10 +63,14 @@ You can now use the Firebase `_auth` to authenticate in your Dart code, e.g.
...
   GoogleSignInAccount googleUser = await _googleSignIn.signIn();
   GoogleSignInAuthentication googleAuth = await googleUser.authentication;
-  FirebaseUser user = await _auth.signInWithGoogle(
+
+  final AuthCredential credential = GoogleAuthProvider.getCredential(
     accessToken: googleAuth.accessToken,
     idToken: googleAuth.idToken,
   );
+
+  FirebaseUser  user = await _fbauth.signInWithCredential(credential);
```
Comment (Crazywater) [inline packages/firebase_auth/README.md]: This should probably remain "_auth"

Comment (Crazywater) [inline packages/firebase_auth/README.md]: Tiny nit: Double space

File: packages/firebase_auth/README.md [hunk 2]
```diff
@@ -63,10 +63,14 @@ You can now use the Firebase `_auth` to authenticate in your Dart code, e.g.
 Future<FirebaseUser> _handleSignIn() async {
   GoogleSignInAccount googleUser = await _googleSignIn.signIn();
   GoogleSignInAuthentication googleAuth = await googleUser.authentication;
-  FirebaseUser user = await _auth.signInWithGoogle(
+
+  final AuthCredential credential = GoogleAuthProvider.getCredential(
```
Comment (Crazywater) [inline packages/firebase_auth/README.md]: The other local variables aren't final here either, so I'd leave out final.

**Pull request closed without merging**